00_source_data/data.py

```diff
@@ -101,7 +102,10 @@ def constant_states(states, num_constant_states, df_selection):
 # Generating constant states
 states = ["Florida", "Washington", "Texas"]
 num_constant_states = 10
```

Comment:
I think you changed to 12 below, isnt't it? We maybe do not need these lines of codes anymore

Reply:
Hi @Emmawang00 , that number depends on how many states you all have been working with. If you all have been working with files made from constant states 10, I would have to change it back to 10, so it doesn't conflict with everybody's work.

Reply:
Hi @EricR401S , yes that makes sense;) now we should change all the number of constant_states func to 10;)

Reply:
Hey @Emmawang00 , yeah, I'll have it all set up on the next push I make, I made a note of that here on Visual Studio.

Reply:
They will go back to 10, not 12.